Riot/AppDelegate.m

```diff
@@ -2038,6 +2040,21 @@ - (BOOL)handleUniversalLink:(NSUserActivity*)userActivity
     // iOS Patch: fix vector.im urls before using it
     webURL = [Tools fixURLWithSeveralHashKeys:webURL];
 
+    // Extract required parameters from the link
+    NSArray<NSString*> *pathParams;
+    NSMutableDictionary *queryParams;
+    [self parseUniversalLinkFragment:webURL.absoluteString outPathParams:&pathParams outQueryParams:&queryParams];
+
+    //  Storing now as if it's a supported url
+    //  If it's not, it will be nilled again after this method finishes.
```

Comment:
I do not see it set to nil afterwards. It works for me to keep it (and remove the comment)

Reply:
I removed the nillation, but forgot this.